src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/AddPackageReferenceCommand.cs

```diff
@@ -83,8 +83,8 @@ public static class AddPackageReferenceCommand
                     var packageDependency = new PackageDependency(id.Values[0], VersionRange.Parse(packageVersion));
                     var packageRefArgs = new PackageReferenceArgs(projectPath.Value(), packageDependency, logger)
                     {
-                        Frameworks = MSBuildStringUtility.Split(frameworks.Value()),
-                        Sources = MSBuildStringUtility.Split(sources.Value()),
+                        Frameworks = frameworks.Values.ToArray(),
+                        Sources = sources.Values.ToArray(),
```

Comment:
will this null ref if its an empty value?

Reply:
.Values returns an empty list